src/OmniSharp.Abstractions/Options/CodeActionOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OmniSharp.Options
+{
+    public class CodeActionOptions
+    {
+        public string[] LocationPaths { get; set; }
```

Comment:
it's a bit unfortunate, yes. The reason for this is that this class (`CodeActionOptions`) is used to bind to the ASP.NET Configuration which mandates the presence of a public setter. So I thought leaving it as is and adding a public method that would normalize the paths would be a lesser evil than having another abstraction layer wrapping this or (sic!) a property with setter only to which the config would bind